assets/RNAseq_report.Rmd

```diff
@@ -2009,12 +2038,14 @@ if (isProvided(params$custom_gmt)){
         database_string <- paste0(database_string, "`", db, ": ", gsub("\n", ", ", database_version_info$sources[[db]]$version), "`, ")
     }
     database_string <- paste0(strsplit(database_string, ",\\s*(?=[^,]+$)", perl=TRUE)[[1]], ".")
+    #TODO: is edgeR still relevant? It does not even exist in the env
```

Comment:
It was previously used for the heatmap matrix. If no longer used, it can be removed.